src/shared/2021_robot_constants.cpp

```diff
@@ -10,20 +10,21 @@ RobotConstants_t create2021RobotConstants(void)
         .robot_radius_m  = ROBOT_MAX_RADIUS_METERS,
         // TODO (#2112): update this
```

Comment:
Not sure if these values are all up to date now, but we should probably remove the todo for the ones that are up to date, and if there are any that are still estimates which need to be updated, we should probably not resolve #2112.